### DISCUSSION TITLE
新增的文章在QEXO里面不显示

### DISCUSSION THREAD

**Discussion by miaoyanginfo** [Qexo/Qexo#578] [General]
我用其他编辑器发布的文章，在QEXO里面不显示。站点实际有59篇，QEXO里面只有57篇。

**Comment by am-abudu** [comment]
使用的是什么博客框架，请检查对应的文章目录是否有相应文件

**Reply by miaoyanginfo** [reply]
hexo。目录内有文件。这两文件是在装完QEXO之后用obsidian发布的，是否和这个有关系

**Reply by am-abudu** [reply]
请阅读 [文档](https://www.oplog.cn/qexo/start/questions.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E6%88%91%E6%96%B0%E5%BB%BA%E7%9A%84%E6%96%87%E7%AB%A0%E6%B2%A1%E6%9C%89%E6%98%BE%E7%A4%BA-qexo-%E5%A6%82%E4%BD%95%E4%BF%9D%E8%AF%81%E6%96%87%E4%BB%B6%E6%9C%80%E6%96%B0) 以了解详情

**Comment by miaoyanginfo** [comment]
收到，我设置后观察下效果，感谢。
